### PR TITLE
Get transactions by polkascan

### PR DESCRIPTION
**Pull request by imstar15**
image

### CODE REVIEW

File: packages/landing/src/common/polkascan/index.js [hunk 1]
```diff
@@ -2,54 +2,82 @@ import _ from 'lodash';
...
-				attributes.params = params;
-				attributes.datetime = datetime;
-			}
-		});
-		return extrinsics;
+		try {
+			const url = `${BASE_URL}/extrinsic?filter[signed]=1&filter[module_id]=${moduleId}&filter[call_id]=${callId}&page[size]=${PAGE_SIZE}`;
+			const response = await fetch(url);
+			const responseData = await response.json();
+			let extrinsics = responseData.data;
```
Comment (chrisli30) [inline packages/landing/src/common/polkascan/index.js:22]: I would add a check to filter out cases such as responseData.data = "undefined" or "not a array"

Also, you need to check responseData with `responseData && responseData.data`

```
if(_.isEmpty(responseData && responseData.data))
{
return [];
}
```

File: packages/landing/src/common/polkascan/index.js [hunk 2]
```diff
@@ -2,54 +2,82 @@ import _ from 'lodash';
 import moment from 'momnet';
 const BASE_URL = 'http://rpc.testnet.oak.tech:8080/api/v1';
 const PAGE_SIZE = 10000;
-const DECIMAL = 10e-11;
+const DECIMALS = 10e9;
```
Comment (chrisli30) [inline packages/landing/src/common/polkascan/index.js:5]: Why this is 10e9? Add comment.

Isn't OAK's token decimal 10e10?

File: packages/landing/src/common/polkascan/index.js (hunk 1 above)
Comment (imstar15) [reply packages/landing/src/common/polkascan/index.js:27]: Fixed

File: packages/landing/src/common/polkascan/index.js (hunk 2 above)
Comment (imstar15) [reply packages/landing/src/common/polkascan/index.js]: Fixed

File: packages/landing/src/config.js [hunk 3]
```diff
@@ -2,10 +2,16 @@ const config = {
...
+  // polkascan
+  polkascan: {
+    baseUrl: 'http://rpc.testnet.oak.tech:8080/api/v1',
+  },
+
   // OAK token info
   oak: {
     symbol: 'OAK',
     price: 0.35,
+    decimals: 10,
```
Comment (chrisli30) [inline packages/landing/src/config.js:14]: Rename `decimals` to `decimal` as it is a number but not a plural.

**Pull request merged**